internal/clients/idler.go

```diff
@@ -9,7 +9,6 @@ import (
 
 	"github.com/fabric8-services/fabric8-jenkins-proxy/internal/util"
 	"github.com/fabric8-services/fabric8-jenkins-proxy/internal/util/logging"
-	"github.com/pkg/errors"
```

Comment:
This seems troubling. your changes to kishansagathiya/fabric8-jenkins-proxy with a seperate commit just for UI changes?